crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1189,6 +1192,16 @@ impl PgManager {
         Ok(Some(domain.to_string()))
     }
 
+    pub(crate) async fn fetch_sui_system_state(
+        &self,
+        epoch_id: u64,
+    ) -> Result<NativeSuiSystemStateSummary, Error> {
+        Ok(self
+            .inner
+            .spawn_blocking(move |this| this.get_epoch_sui_system_state(Some(epoch_id)))
```

Comment:
I feel like we've been swinging back and forth between using IndexerReader methods or defining query patterns in the _backend.rs files ... we should standardize

Reply:
Which one do you suggest? I mostly followed what we had.

Reply:
I think this is what we have currently right? So I guess it's fine for now

Reply:
We should probably revisit this and change accordingly everywhere.